Demo/Application/Features/PayPal - Checkout (Native UI)/BraintreeDemoPayPalNativeUIViewController.swift

```diff
@@ -0,0 +1,58 @@
+import UIKit
+
+class BraintreeDemoPayPalNativeUIViewController: BraintreeDemoPaymentButtonBaseViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        let textView = UITextView()
+        textView.translatesAutoresizingMaskIntoConstraints = false
+        textView.textAlignment = .center
+        textView.text = "This demo is using a hard-coded tokenization key for a BT sandbox with a linked PayPal sandbox account."
+        self.view.addSubview(textView)
+
+        NSLayoutConstraint.activate([
+            textView.widthAnchor.constraint(equalTo: view.widthAnchor),
+            textView.topAnchor.constraint(equalTo: self.paymentButton.bottomAnchor),
+            textView.heightAnchor.constraint(equalToConstant: 60.0)
+        ])
+    }
+
+    override func createPaymentButton() -> UIView? {
+        let button = UIButton(type: .custom)
+        button.setTitle(NSLocalizedString("PayPal Checkout (Native UI)", comment: ""), for: .normal)
+        button.setTitleColor(UIColor.blue, for: .normal)
+        button.setTitleColor(UIColor(red: 50.0 / 255, green: 50.0 / 255, blue: 255.0 / 255, alpha: 1.0), for: .highlighted)
+        button.setTitleColor(UIColor.lightGray, for: .disabled)
+        button.addTarget(self, action: #selector(tappedPayPalCheckout(_:)), for: .touchUpInside)
+        return button
+    }
+
+    @objc func tappedPayPalCheckout(_ sender: UIButton?) {
+        self.progressBlock("Tapped PayPal Checkout (Native UI) - using BTPayPalDriver")
+
+        sender?.setTitle(NSLocalizedString("Processing...", comment: ""), for: .disabled)
+        sender?.isEnabled = false
+
+        // This demo uses a hard coded tokenization key.
+        // You can find the corresponding accounts for "PayPal Native Checkout" in 1Password
+        // TODO: obtain credentials for a BT-PP linked prod account. Update this demo to properly toggle b/w envs.
+        let apiClient = BTAPIClient.init(authorization: "sandbox_8hpbq6nh_ks3tvzmhdjryvknn")
+        let driver = BTPayPalDriver(apiClient: apiClient!)
+        let request = BTPayPalCheckoutRequest(amount: "4.30")
+        request.useNativeUI = true
```

Comment:
This is out of scope for this PR, but wouldn't `prefersNativeUI` be a better name on `PayPalCheckoutRequest` since not all merchants will have the ability to use Native XO initially?

Reply:
Hm - why wouldn't merchants have the ability to use Native XO initially?

Reply:
So - all existing Braintree merchants that can accept PayPal will already need linked BT - PP merchant control panels, for prod (there is a chance that they need to set this up for Sandbox and are using our mocked sandbox flow).

But yeh - we can definitely revisit the naming of this property and hopefully make it consistent with Android as best we can. Let's definitely continue the convo, but I am going to merge this PR for now 👍